apps/emqx_durable_storage/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -282,8 +282,13 @@ shard_info(DB, Shard) ->
         [#?SHARD_TAB{replica_set = Replicas}] ->
             ReplicaSet = maps:from_list([
                 begin
-                    %% TODO:
-                    ReplInfo = #{status => up},
+                    Status =
+                        case mria:cluster_status(?MODULE:node(I)) of
```

Comment:
This will tell status of the site rather than DB shard. This is good for now, but please add a follow-up JIRA to make this check more DB specific in the future.

Reply:
:checkered_flag: https://emqx.atlassian.net/browse/EMQX-12564